### PR TITLE
DDF-04264 updates lodash and plotly to newer versions

### PR DESCRIPTION
**Pull request by stustison**
#### What does this PR do?
Updates lodash and plotly to newer versions
#### Who is reviewing it? 
@djblue @garrettfreibott 

#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@stustison

#### How should this be tested?
Launch all of the effected UIs
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4264 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "moment-timezone": "0.5.16",
     "mt-geo": "1.0.1",
     "openlayers": "4.6.5",
-    "plotly.js": "1.10.5",
```
Comment (andrewkfiedler) [inline ui/packages/catalog-ui-search/package.json]: I remembered there being a reason for not updating plotly, and I'm not sure if it still applies: https://github.com/codice/ddf/pull/1610

> Downgrades plotly dependency to address security issues. Upgrading to address is not possible until they address this issue: plotly/plotly.js#895

So whoever tests this should ensure to try out the histogram and verify it works as expected in IE11.

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Pull request merged**